### DISCUSSION TITLE
Integrate github with slack.

### DISCUSSION THREAD

**Discussion by cdsre** [chaostoolkit/chaostoolkit#305] [Ideas]
In my spare time I visit a number of the chaostoolkit repositories checking for any new issues raised. This can be a bit time consuming and it's not always at the front of my mind. 

If i had a means to receive notifications or a central place to review all issues I would be able to proactivly loom at these in more realt time. As far as I know github doesn't provide a feature to allow me to automatically be notified of new issues in the projects.

Can we integrate github to the slack space. Maybe create a channel for issues in all projects, or a pull requests channel. This would allow us to have a single source for all the repos

You can read more here https://github.com/integrations/slack?tab=readme-ov-file#features

**Comment by Lawouach** [comment]
Good call Chris.

I had created such a channel long ago but kept it private. Not all repositories are plugged into it however. So we'll have to connect as we go along.

I've invited you to the channel on slack.
